src/clj/puppetlabs/services/puppet_admin/puppet_admin_service.clj

```diff
@@ -22,22 +23,9 @@
       (cond
         (or (false? (:authorization-required whitelist-settings))
             (not-empty client-whitelist))
-          (log/warn
-           "The 'client-whitelist' and 'authorization-required' settings in"
-           "the 'puppet-admin' section are deprecated and will be removed"
-           "in a future release.  Remove these settings and create an"
-           "appropriate authorization rule in the"
-           "/etc/puppetlabs/puppetserver/conf.d/auth.conf file.")
+        (log/warn (trs "The ''client-whitelist'' and ''authorization-required'' settings in the ''puppet-admin'' section are deprecated and will be removed in a future release.  Remove these settings and create an appropriate authorization rule in the /etc/puppetlabs/puppetserver/conf.d/auth.conf file."))
```

Comment:
Let's split these sentences up into separately translated strings...  as this is reused from the previous file, and used on line 28.